aws/src/main/java/org/apache/iceberg/aws/s3/signer/S3V4RestSignerClient.java

```diff
@@ -338,6 +339,7 @@ public SdkHttpFullRequest sign(
       Consumer<Map<String, String>> responseHeadersConsumer = responseHeaders::putAll;
       S3SignResponse s3SignResponse =
           httpClient()
+              .withAuthSession(org.apache.iceberg.rest.auth.AuthSession.EMPTY)
```

Comment:
I believe this change can be reverted now that the session is set in `httpClient()`